atm.js

```diff
@@ -1,21 +1,71 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$(document).ready(function(){
+  var checking = new Account ($("#checking"));
+  var savings = new Account($("#savings"))
+});
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+function Account(element){
+  var account =  this;
+  account.element = element;
+  account.balance = 0;
+  account.elements = {
+    deposit: account.element.find(".deposit"),
+    withdraw: account.element.find(".withdraw"),
+    balance: account.element.find(".balance"),
+    input: account.element.find(".input")
+  }
+  account.elements.deposit.click(function(){
+    var input = parseInt(account.elements.input.val());
+    account.balance += input;
+    account.elements.balance.html("$" + account.balance);
+  })
+  account.elements.withdraw.click(function(){
+    var input = parseInt(account.elements.input.val());
+    account.balance -= input;
+    account.elements.balance.html("$" + account.balance);
```

Comment:
Next step: how can you prevent withdraw from being executed when the amount-to-be-withdrawn is greater than the current account balance?
* What two values would you need to compare? How can you reference them?
* Assuming withdraw > deposit, what Javascript features could you use to prevent any modification to the balance (both the data and its representation in the DOM)?